### DISCUSSION TITLE
What is the best way to start contributing to this repo?

### DISCUSSION THREAD

**Discussion by aaryan-gulia** [ripple/rippled#4930] [Q&A]
I'm a C++ developer excited about the language's applicability to the blockchain space. Consequently, this repository intrigued my interest. Are there any best practices you would recommend if I want to start contributing to this codebase?

**Comment by ximinez** [comment]
https://github.com/XRPLF/rippled/blob/develop/CONTRIBUTING.md

**Comment by mvadari** [comment]
https://github.com/XRPLF/rippled/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22

**Comment by elyosemite** [comment]
@aaryan-gulia me too. I have been reading a lot about this project lately and I've been loving it. I started with CONTRIBUTION mentioned by @ximinez.
